src/main/java/org/kainos/ea/controller/JobSpecController.java

```diff
@@ -0,0 +1,23 @@
+package org.kainos.ea.controller;
+
+import io.swagger.annotations.Api;
+import javax.ws.rs.Path;
+import org.kainos.ea.service.JobSpecService;
+import org.kainos.ea.utility.DatabaseConnector;
+import org.kainos.ea.validator.JobSpecValidator;
+
+@Api("Kainos Job Application")
+@Path("/job-specification")
+public class JobSpecController {
+
+  private static JobSpecService jobSpecService;
+  private static JobSpecValidator jobSpecValidator;
+
+  public JobSpecController() {
+    DatabaseConnector databaseConnector = new DatabaseConnector();
+//    jobSpecService = new JobSpecService(new JobSpecDao(), databaseConnector);
+    jobSpecValidator = new JobSpecValidator();
+  }
+
+
```

Comment:
Does spotless include the ability to check for extra white lines?

Reply:
<img width="887" alt="image" src="https://github.com/leeservice/as-team1-api/assets/137312132/62bad5e6-96c9-4f35-8bfd-8225b74f5622">
It does seem to fix extra lines both in methods and outside of them, not sure why these get through at times

Reply:
It might be a configuration thing where it's happy with 2 lines. That's grand